ballerina/annotation.bal

```diff
@@ -16,11 +16,13 @@
 
 # Configuration for a WebSub Hub service.
 #
-# + leaseSeconds - The period for which the subscription is expected to be active in the `hub`
-# + webHookConfig - HTTP client configurations for subscription/unsubscription intent verification
+# + leaseSeconds - The period for which the subscription is expected to be active in the `hub`  
+# + webHookConfig - HTTP client configurations for subscription/unsubscription intent verification  
+# + autoVerifySubscription - Configuration to enable or disable automatic subscription verification
 public type ServiceConfiguration record {|
     int leaseSeconds?;
     ClientConfiguration webHookConfig?;
+    boolean autoVerifySubscription = false;
```

Comment:
As per Maryam's suggestion, are we going to change this to `autoIntentVerification` and something similar?

Reply:
+1

Reply:
+1. this is more clearer

Reply:
Fixed with [a0a77029ab09336aeda3c5cedacde8205622abd6](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/1083/commits/a0a77029ab09336aeda3c5cedacde8205622abd6)